### ISSUE TITLE
what is hyper-parameters for the best checkpoint?

### ISSUE THREAD

**Issue by zplovekq** [krasserm/fairseq-image-captioning#10]
I sincerely want to know the hyper-parameters for the checkpoint20 and checkpoint24.
Thanks!

**Comment by krasserm** [comment]
Checkpoint 20 was obtained from [this training run](https://github.com/krasserm/fairseq-image-captioning#cross-entropy-loss) and checkpoint 24 from [this one](https://github.com/krasserm/fairseq-image-captioning#self-critical-sequence-training). Both were obtained from parallel training on two GTX 1080 GPUs.

**Comment by zplovekq** [comment]
sincerely thanks for your reply!
I have tried these hyper-parameters as you said.I train on 3 RTX 2080Ti and I get BELU 33.9 with the score.sh in your repository. So if these is some difference in different GPU?
Thanks!

**Comment by krasserm** [comment]
BLEU-4 33.9 after CE loss training or after SCST training?

**Comment by zplovekq** [comment]
For the CE loss :D

**Comment by zplovekq** [comment]
![image](https://user-images.githubusercontent.com/39880124/77246470-8c05b300-6c62-11ea-8781-80cd1b11ead9.png)
I trained for about 100 epoch and the fairseq automaticly save the checkpoint best for the valid subset.
I use the  generate.py in your repository to generate test-prediction.json and use it for score.sh
Thanks!

**Comment by krasserm** [comment]
You additionally need to fine-tune with SCST to get BLEU-4 scores up to 39.

**Comment by zplovekq** [comment]
Hi. I use the SCST to train with the train command in your README.md
And i have tried a few times, all the training loops are str I use the SCST to train with the train command in your README.md
And i have tried a few times, all the training loops are struck in the 41%, like this:
![image](https://user-images.githubusercontent.com/39880124/77419780-ebe78f80-6e03-11ea-95fc-951080574242.png)
then the  loop stop.
I didn't know why.Is there I do something wrong?
Thanks!

**Comment by FantasyoO666** [comment]
> 

Hi, my loss is alaways be -0 or +0, please tell me this is why？thank you！
